packages/react-native/android/src/oldarch/HotUpdaterModule.kt

```diff
@@ -29,18 +32,20 @@ class HotUpdaterModule internal constructor(
         zipUrl: String?,
         promise: Promise,
     ) {
-        val isSuccess =
-            HotUpdater.updateBundle(mReactApplicationContext, bundleId, zipUrl) { progress ->
-                val params =
-                    WritableNativeMap().apply {
-                        putDouble("progress", progress)
-                    }
-
-                this.mReactApplicationContext
-                    .getJSModule(DeviceEventManagerModule.RCTDeviceEventEmitter::class.java)
-                    .emit("onProgress", params)
-            }
-        promise.resolve(isSuccess)
+        (currentActivity as? FragmentActivity)?.lifecycleScope?.launch {
+            val isSuccess =
+                HotUpdater.updateBundle(mReactApplicationContext, bundleId, zipUrl) { progress ->
```

Comment:
did you wrapping withContext ?

val isSuccess = withContext(Dispatch.IO~~
                                updateBundle